infrastructure.md

```diff
@@ -38,48 +40,87 @@ goes wrong with the bot.
 It is generated by [rust-dxr](https://github.com/nrc/rust-dxr)
 
-## Others
-
-TODO: Describe these better.
-
-- [perf.rust-lang.org](http://perf.rust-lang.org/). Maintained by nrc.
-  [Source](https://github.com/nrc/rustc-perf).
-- [Rust Release Explorer](http://ashleygwilliams.github.io/rust-release-explorer/)
-- [rusty-dash.com](http://rusty-dash.com/).
-  [Source](https://github.com/dikaiosune/rust-dashboard).
-- [buildbot.rust-lang.org](http://buildbot.rust-lang.org/).
-  [Source](https://github.com/rust-lang/rust-buildbot). Maintained by acrichto.
-- [crater.rust-lang.org](https://crater.rust-lang.org/).
-  [Source](https://github.com/rust-lang-nursery/crater). Maintained by
-  tomprince.
-- [play.rust-lang.org](http://play.rust-lang.org/)
-  [Source](https://github.com/integer32llc/rust-playground). Maintained by
-  shepmaster.
-
-[rust-central-station]: https://github.com/alexcrichton/rust-central-station
+## perf
+
+[perf](https://github.com/rust-lang-nursery/rustc-perf) offers information
+about the performance of `rustc` over time.
+
+It is split into a data [collector] and a [web frontend]. The raw performance
+data is available [here](https://github.com/rust-lang-nursery/rustc-timing)
+and can be browsed on the [perf website](https://perf.rust-lang.org).
+
+[collector]: https://github.com/rust-lang-nursery/rustc-perf/tree/master/collector
+[web frontend]: https://github.com/rust-lang-nursery/rustc-perf/tree/master/site
+
+## Rust Playground
+
+[Rust Playground](https://github.com/integer32llc/rust-playground) allows you
+to experiment with Rust before you install it locally, or in any other case
+where you might not have the compiler available. The Rust playground can be
+accessed [here](https://play.rust-lang.org).
+
+## Crater
+
+[Crater](https://github.com/rust-lang/crater) is a tool to run
+experiments across the whole Rust ecosystem. Its primary purpose is to detect
+regressions in the Rust compiler, and it does this by building large number of
+crates, running their test suites and comparing the results between two
+versions of the Rust compiler.
+
+## docs.rs
+
+[docs.rs](https://docs.rs/) builds and serves the rustdoc documentation for
+all crates on [crates.io]. Issues may be filed on the [docs.rs
+repository](https://github.com/rust-lang/docs.rs). See the #docs-rs channel on
+Discord for discussion or urgent issues.
+
+## Rust Log Analyzer
+
+The [Rust Log Analyzer](https://github.com/rust-lang/rust-log-analyzer)
+analyzes CI build logs to extract error messages and posts them to the pull
+request. It is run by [TimNN](https://github.com/TimNN).
+
+## Toolstate
+
+The state of tools included with Rust are tracked on the [toolstate
+page](https://rust-lang-nursery.github.io/rust-toolstate/). When each PR is
+merged via CI, the status of each tool is recorded in a JSON file and stored
+in the [toolstate repo](https://github.com/rust-lang-nursery/rust-toolstate).
```

Comment:
Say something about the toolstate week and when it happens? cc @pietroalbini

Reply:
Toolstate is being documented in #243.  This can link to that if/when that gets merged.

Reply:
Alright; sounds good.

Reply:
I added a link from here to the new page in #243.